src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AzureUserService.java

```diff
@@ -32,47 +34,51 @@
     @Autowired
     private UserConfiguration userConfiguration;
 
+    @Autowired
+    private ClientConfiguration clientConfiguration;
+
     /**
-     * Creates a new subscriber in the Azure active directory.
-     * @param subscriber The subscriber to add in the azure active directory.
+     * Creates a new azureAccount in the Azure active directory.
+     * @param azureAccount The azureAccount to add in the azure active directory.
      * @return The created user if it was successful.
      * @throws AzureCustomException thrown if theres an error with communicating with Azure.
      */
-    public User createUser(Subscriber subscriber) throws AzureCustomException {
+    public User createUser(AzureAccount azureAccount) throws AzureCustomException {
         try {
-            User user = createUserObject(subscriber);
+            User user = createUserObject(azureAccount);
             return graphClient.users()
                 .buildRequest()
                 .post(user);
         } catch (GraphServiceException e) {
             LOGGER.error(e.getMessage());
```

Comment:
we already have a logger with Slf4j, is this creating a separate logger?

Reply:
Ah yeah, it was using the old style creation rather than the annotation. I've actually removed this from here and instead put it in the AccountService as it felt better in there.